cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -380,17 +353,18 @@ def test_service_ansible_playbook_plays_table(service_catalog, service_request,
         initialEstimate: 1/6h
         tags: ansible_embed
     """
-    service_catalog.order()
-    service_request.wait_for_request()
-    view = navigate_to(service, "Details")
+    ansible_service_catalog.order()
```

Comment:
The ansible_service fixture will order the catalog item for you right ?
If yes then it can be used so service is deleted after fixture use.

Reply:
Yes, it will order catalog item for me, but it won't delete catalog item, it will only retire that service.
@sshveta Anything else you wanted to suggest here .?